selfdrive/thermald/thermald.py

```diff
@@ -156,6 +158,24 @@ def handle_fan_uno(max_cpu_temp, bat_temp, fan_speed, ignition):
   return new_speed
 
 
+def set_offroad_alert_if_changed(offroad_alert, show_alert):
+  if offroad_alert in prev_offroad_states and \
```

Comment:
```suggestion
  if prev_offroad_states.get(offroad_alert, None) == show_alert:
```

Reply:
I guess your version is more explicit/pythonic. You can choose what you like better.